lib/shelley/src/Cardano/Wallet/Shelley/Network/Blockfrost.hs

```diff
@@ -161,10 +195,13 @@ withNetworkLayer tr net project k = k NetworkLayer
     , stakeDistribution = undefined
     , getCachedRewardAccountBalance = undefined
     , fetchRewardAccountBalances = undefined
-    , timeInterpreter = undefined
+    , timeInterpreter = timeInterpreter getGenesisBlockDate
```

Comment:
The name clash between field `timeInterpreter` and function `timeInterpreter` could be confusing. 🤔  I don't really mind myself, but perhaps adopting the style from `Cardano.Wallet.Shelley.Network.Node` and using an underscore prefix `_timeInterpreter` could prevent accidental confusion later on?

Reply:
I don't think we should go as far as renaming (`_`-ing) accessor function(s) now (its asks for a project-wide consensus on naming, might involve discussing lenses etc). I'll give the `timeInterpreter` function some other name.